ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -89,6 +89,16 @@ Item {
             chatSectionModule: root.rootStore.chatCommunitySectionModule
         }
 
+        readonly property string linkPreviewEnabledNotification: `%1<a style="text-decoration:none" href="#${Constants.appSection.profile}/${Constants.settingsSubsection.messaging}">%2</a>.`
+                                                                    .arg(qsTr("Link previews will be shown for all sites. You can manage link previews in "))
+                                                                    .arg(qsTr("Settings"))
+        readonly property string linkPreviewDisabledNotification: `%1<a style="text-decoration:none" href="#${Constants.appSection.profile}/${Constants.settingsSubsection.messaging}">%2</a>.`
+                                                                    .arg(qsTr("Link previews will never be shown. You can manage link previews in "))
+                                                                    .arg(qsTr("Settings"))
+        readonly property string linkPreviewEnabledForMessageNotification: `%1<a style="text-decoration:none" href="#${Constants.appSection.profile}/${Constants.settingsSubsection.messaging}">%2</a>.`
```

Comment:
`%1<a style="text-decoration:none" href="#${Constants.appSection.profile}/${Constants.settingsSubsection.messaging}">%2</a>.`
is reused 3 times, maybe create a constant for it?